doc/rdt.md

```diff
@@ -36,7 +36,9 @@ resources. Resources include portions of caches (L2 and L3) and memory
 bandwidth (MB). Cache partitioning is exclusive: cache portions of two
 partitions are not allowed to overlap. However, by design of the underlying
 technology, MB allocations are not exclusive. Thus, it is possible to assign
-all partitions 100% of memory bandwidth, for example.
+all partitions 100% of memory bandwidth, for example. `Partition` is a concept 
```

Comment:
I would rephrase this as something like:

"""
Partitions are purely a `goresctrl` concept. They are designed to help dividing resources managed by the Linux `resctrl` filesystem into disjoint sets that do not overlap, yet collectively fully utilize the set of available resources. Partitions do not have a corresponding abstraction in the Linux resctrl filesystem itself.
"""